src/__test__/mocks.ts

```diff
@@ -32,6 +32,12 @@ export const mockExpandedBeerStyle: BJCPStyle = {
   },
   preamble: 'important context about this style',
   entry_instructions: 'specify carbonation level',
+  examples: {
+    group: [
+      { '@_label': 'Dark', '#text': 'One, Two' },
+      { '@_label': 'Pale', '#text': 'Uno, Dos' },
+    ]
+  }
```

Comment:
An alternative approach would be:

```xml
<group>
  <label></label>
  <examples></examples>
</group>
```

...but then the JSON is like `examples.group[].label` and `examples.group[].examples` which 🤮

Reply:
Oh ... and doing:

```xml
<examples>
  <group></group>
</examples>
```

(i.e., an unlabeled group) for ALL the styles produced `examples.group` not `examples.group[]` ... so unless I want to throw another transformation into the ETL script. 😞

Reply:
> so unless I want to throw another transformation into the ETL script. 😞

...which [I'm already doing for `range`](https://github.com/founddrama/bjcp-viewer/blob/main/src/bjcp/bjcp-xml-to-json.js#L21,L34) so maybe that's not so bad?